### ISSUE TITLE
List of Evaluation Tools - Need page headings

### ISSUE THREAD

**Issue by MicheleAWilliams-A11y** [w3c/wai-eval-tools#115]
The page is missing subheadings to describe the content. This is necessary for WCAG (SC 2.4.6), and I suggest using a design similar to the Course List: https://github.com/w3c/wai-course-list/issues/279#issuecomment-1196420125

**Comment by MicheleAWilliams-A11y** [comment]
Opened in the wrong repository
